### PR TITLE
Resturant Page

### PR DESCRIPTION
**Pull request by GraceOyiza**
In this project, I built a simple webpage for a restaurant. We have :
- Home page
- Menu Page
- Contact Page

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -0,0 +1,49 @@
+{
+  "name": "resturant_page",
+  "version": "1.0.0",
+  "description": "\"\"",
+  "main": "index.js",
+  "scripts": {
+    "build": "webpack --watch"
+  },
```
Comment (addod19) [inline package.json]: Kindly improve the script, use this as a guide 
``` "scripts": {
    "dev": "webpack --mode development",
    "build": "webpack --mode production",
    "start": "webpack-dev-server --mode development --open" // optional
  }
```

**Review by addod19 — CHANGES_REQUESTED** [review]
## Status: Require changes :recycle: :traffic_light: 

Hi @GraceOyiza, congratulations on this project :muscle: The following requirements have been met:
- Great use of DOM manipulations :+1: and JS object literals
- All modules use ES6 syntax and creates a div element

Require changes below:
- Kindly add webpack-dev-server to the package.json file under the devDependencies
- Kindly improve the script so we can run commands like; `npm run dev` `npm run build`

Once these are resolved :+1: Kindly submit another code review request
_Happy coding_ :woman_technologist: 
[Daniel Larbi Addo](https://www.linkedin.com/in/daniel-larbi-addo/)

**Comment by GraceOyiza** [comment]
- All required changes fixed.

**Review by Torres-ssf — APPROVED** [review]
### PR Approved :+1: 
Hi @GraceOyiza :hand: 
Great job on this project.
You may merge the PR.
Good luck with the next one.

Happy coding :computer: 
**Sérgio Torres**

**Pull request merged**